src/components/App.js

```diff
@@ -1,14 +1,29 @@
-import React from 'react';
-import Display from './Display';
+import React, { useState } from 'react';
 import ButtonPanel from './ButtonPanel';
 import calculate from '../logic/calculate';
+import Display from './Display';
+
+const App = () => {
```

Comment:
- [ ] [REQUIRED] it is converted to a stateful component using class ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone5/app/stateful.png))
- [ ] [REQUIRED] it has state initialized in constructor ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone5/app/stateful.png))